platformio.ini

```diff
@@ -12,6 +12,8 @@
 platform = teensy
 board = teensy41
 framework = arduino
+build_unflags = -std=gnu++11
+build_flags = -std=c++17
```

Comment:
what is this for?

Reply:
specify c++17 as the std to use. some funcs and libs don't exist in older versions